db/migrate/20160718114501_update_existing_builtin_to_boolean1_in_roles.rb

```diff
@@ -0,0 +1,9 @@
+class UpdateExistingBuiltinToBoolean1InRoles < ActiveRecord::Migration
+  def up
+    roles = Role.where(:builtin => 2)
+    roles.each do |role|
+      role.builtin = 1
+      role.save!
```

Comment:
SQL would be better here.
`ActiveRecord::Base.connection.execute("UPDATE roles SET builtin = 1 where builtin = 2")`